src/client/menu.c

```diff
@@ -1,20 +1,19 @@
 #include <string.h>
 #include <stdio.h>
 #include <SDL_FontCache.h>
-#include <SDL_image.h> 
-#include <SDL_mixer.h>
 
-#include "menu.h"
-#include "game.h"
-#include "fonts.h"
-#include "gamenet.h"
-#include "audio.h"
 
-void LoadMenu(Menu* menu){
-    Game* game = GetGame();
+void LoadMenu(Menu *menu)
```

Comment:
Consider using a GetMenu instead of passing as parameter
Don't store an instance of Menu in Game